projects/spectator/jest/src/host.ts

```diff
@@ -1,31 +1,9 @@
-/**
- * @license
- * Copyright Netanel Basal. All Rights Reserved.
- *
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://github.com/NetanelBasal/spectator/blob/master/LICENSE
- */
-
 import { Type } from '@angular/core';
-import { SpectatorWithHost as BaseSpectatorWithHost, createHostComponentFactory as baseCreateHostComponentFactory, HostComponent, SpectatorOptions, isType, Token } from '@netbasal/spectator';
-
-import { mockProvider, SpyObject } from './mock';
-
-export class SpectatorWithHost<C, H = HostComponent> extends BaseSpectatorWithHost<C, H> {
-  /**
-   * @inheritDoc
-   */
-  get<T>(type: Token<T>, fromComponentInjector = false): T & SpyObject<T> {
-    return super.get(type, fromComponentInjector) as T & SpyObject<T>;
-  }
-}
+import { createHostComponentFactory as baseCreateHostComponentFactory, HostComponent, SpectatorOptions, SpectatorWithHost as BaseSpectatorWithHost } from '@netbasal/spectator';
+import { CreateComponentOptions } from '../../src/lib/types';
 
-export function createHostComponentFactory<C, H = HostComponent>(options: SpectatorOptions<C, H> | Type<C>): (template: string, detectChanges?: boolean, complexInputs?: Partial<C>) => SpectatorWithHost<C, H> {
-  if (!isType(options)) {
```

Comment:
@NetanelBasal Why did you remove this block? It is needed to create Jest compatible spies (the whole point of extending in this secondary entry point) ;)

Reply:
Because I don't understand why we need it? the `baseCreateHostComponentFactory` should take care of it.

Reply:
It's for the `  (options.mocks || []).forEach(type => options.providers.push(mockProvider(type)));`?

Reply:
Yes

The trick is that this one uses the `mockProvider()` definition of `jest/mock.ts` instead of the base one. :)

Reply:
Sorry, it was late :) You can re-add it.

Reply:
Would be nicer if we could pass this overridden function in a cleaner way,

e.g.

```typescript
function someCreateSpyFunctionForExampleForJest() {}

createHostComponentFactory({
   createSpyObject: someCreateSpyFunctionForExampleForJest
})
```

What do you think?

Reply:
LGTM

Reply:
All right, let me try something 😄

Reply:
Should be fixed now